wallet/src/main/scala/org/bitcoins/wallet/internal/TransactionProcessing.scala

```diff
@@ -272,21 +275,9 @@ private[wallet] trait TransactionProcessing extends WalletLogger {
           insertTransaction(transaction, blockHashOpt)
         else Future.unit
 
-      // unreserved outputs now they are in a block
-      outputsToUse = blockHashOpt match {
```

Comment:
@benthecarman Sorry, this diff was meant to be on this PR

The reason this isn't need is because this gets done in `markAsSpent()` 

https://github.com/bitcoin-s/bitcoin-s/pull/2919/files#diff-fd826a398083437abb5cf5acafa8af283c5420c626aa902469a5b743b8dfa094R328